_unittest/test_utils.py

```diff
@@ -0,0 +1,70 @@
+"""Test utility functions of PyAEDT.
+"""
+
+import logging
+from unittest.mock import MagicMock
+from unittest.mock import patch
+
+import pytest
+
+from pyaedt import settings
+from pyaedt.generic.general_methods import pyaedt_function_handler
+
+SETTINGS_RELEASE_ON_EXCEPTION = settings.release_on_exception
+SETTINGS_ENABLE_ERROR_HANDLER = settings.enable_error_handler
+
+
+@pyaedt_function_handler(deprecated_arg="trigger_exception")
+def foo(trigger_exception=True):
+    """Some dummy function used for testing."""
+    if trigger_exception:
+        raise Exception("Dummy message.")
+
+
+@patch("pyaedt.generic.desktop_sessions._desktop_sessions")
+def test_handler_release_on_exception(mock_sessions):
+    """Test handler while activating/deactivating error handler."""
```

Comment:
```suggestion
    """Test handler while activating or deactivating error handler."""
```
Avoid using a forward slash as it isn't always clear whether it represents an "or" or an "and".